packages/paste-icons/src/react/helpers/IconWrapper.tsx

```diff
@@ -1,18 +1,35 @@
 import styled from '@emotion/styled';
+import {compose, size, system} from 'styled-system';
+import {IconSize, TextColor} from '@twilio-paste/types';
+
+// I can't name this simply `color` because TS + Emotion doesn't like it
+const iconColor = system({
```

Comment:
kind of like how Text needs to accept `textColor` instead of `color`.  Would love if we can figure this out because `color` is more ergonomic